components/sidebar/sidebar_channel/index.js

```diff
@@ -33,7 +35,9 @@ function makeMapStateToProps() {
         const channelId = ownProps.channelId;
 
         const config = getConfig(state);
+        const currentChannelId = getCurrentChannelId(state);
         const channel = getChannel(state, {id: channelId}) || {};
+        const draft = channel.id ? getPostDraft(state, StoragePrefixes.DRAFT, channel.id) : false;
```

Comment:
Having this default to false won't work since the `hasDraft` doesn't check if false is an object before checking draft.message

Reply:
Oh. This works in JavaScript. `false.draft === undefined` 🤔